edu/isi/tiburon/Tiburon.java

```diff
@@ -3143,14 +3143,14 @@ else if (trs1 == null && rs1 == null && tree1 == null)
 		}
 		catch (Exception e) {
 			System.err.println("Throwing generic exception of type "+e.getClass().toString());
-			StackTraceElement elements[] = e.getStackTrace();
-			int n = elements.length;
-			for (int i = 0; i < n; i++) {       
-				System.err.println(e.toString()+": "+elements[i].getFileName() + ":" 
-						+ elements[i].getLineNumber() 
-						+ ">> " 
-						+ elements[i].getMethodName() + "()");
-			}
+//			StackTraceElement elements[] = e.getStackTrace();
```

Comment:
It superfluously prints the exception stack. It makes sense in standalone mode to print it, but we link it as a library, so the exception stack gets printed, eventually by our main. Or the exception gets handled, and no stack trace is necessary.

Basically reduction in spam.